geode-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisServer.java

```diff
@@ -241,7 +241,7 @@ private int setNumWorkerThreads() {
    * to the first non-loopback address
    */
   public GeodeRedisServer() {
-    this(null, -1, null);
+    this(null, 0, null);
```

Comment:
Should this use RANDOM_PORT_INDICATOR instead of 0?

Reply:
Definitely.